src/playground/service.clj

```diff
@@ -73,6 +79,15 @@
     :enter (fn [context]
              (update context :request authentication-request session-auth-backend))}))
 
+(def authorization-interceptor
```

Comment:
`authorization-interceptor` has a generic name, but its implementation compares against a specific role (admin). That's surprising / hard to maintain.

Solve it (I can think of at least two ways)

Reply:
I'd like to also limit the update of an entry to the author and admin by adding the username as a field in each entry. ( this will add some complexity as for an authentication because there will be two different levels. I have a simple (no interceptor solution in the parallel project https://github.com/manandearth/hirundia/blob/a52ed32f605e8fa21a9d0c1a257a1aff96fbf9a7/src/clj/hirundia/services/nests/retrieve/endpoint.clj#L31